engine/uk.co.spudsoft.birt.emitters.excel/src/uk/co/spudsoft/birt/emitters/excel/StyleManagerUtils.java

```diff
@@ -221,7 +222,7 @@ public int poiColumnWidthFromDimension( DimensionType dim ) {
 	 * One of the Font.BOLDWEIGHT_* constants.
 	 */
 	public short poiFontWeightFromBirt(String fontWeight) {
-		if(fontWeight == null || "normal".equals(fontWeight)) {
+		if(fontWeight == null) {

```

Comment:
Was the normal/bold change here intentional? I only ask because your previous birt funtimes was around things being too bold.

Reply:
I actually think the 4_5_0 branch that I'm merging to didn't have my change from my previous work (even though that change what successfully applied to 4.5.0a-9.1 on Nexus, so it looks like the merge I did has picked up that change and it putting it on the 4_5_0 branch with my new changes. The branches feel somewhat arbitrary anyway, given how we are deploying to Nexus manually.

Reply:
We're deploying to nexus manually, but when the next person comes along to fix birt excel they're going to presume that the 4.5 branch they've checked out and are building is correct. If you previous fix for boldening is not in git then the next build will regress.

I'm not sure what has happened with your local git, but based on your previous fix https://planning.greshamtech.com/browse/DWN-26423?focusedCommentId=157846&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-157846 it looks to me that  `"normal".equals(fontWeight)` should be there?

Reply:
I see what happened now. I had a local git reversion for that bold fix (god knows why). That got pulled into my branch when I pushed. I've reverted that reversion now, so this PR shouldn't be taking away that bold fix.